src/util.cpp

```diff
@@ -890,6 +890,20 @@ bool NetworkUtil::ParseNetworkAddress ( QString       strAddress,
 }
 
 
+CHostAddress NetworkUtil::GetLocalAddress()
+{
+    QTcpSocket socket;
+    socket.connectToHost("8.8.8.8", 53); // google DNS, or something else reliable
```

Comment:
This might be critical. The function is a blocking function. If, e.g., the laptop does not have internet access but the server shall be used for local network only, the Jamulus server would not start up.

Reply:
You should not hard-code the IP/port here. Please use a #define and put it in global.h where, e.g., the jamulus.fischvolk.de address is also given.

Reply:
Ah right, OK.

Reply:
Done and tested.  I went with half a second - the target IP used really should respond that fast if the machine doing the look up has adequate internet connectivity.  Otherwise localhost is probably for the best.
02632ea Tune IP look up code